libs/community/langchain_community/utilities/dria_index.py

```diff
@@ -0,0 +1,85 @@
+import logging
+from typing import Any, Dict, List, Optional, Union
+
+logger = logging.getLogger(__name__)
+
+
+class DriaAPIWrapper:
+    """Wrapper around Dria API.
+
+    This wrapper facilitates interactions with Dria's vector search
+    and retrieval services, including creating knowledge bases, inserting data,
+    and fetching search results.
+
+    Attributes:
+        api_key: Your API key for accessing Dria.
+        contract_id: The contract ID of the knowledge base to interact with.
+        top_n: Number of top results to fetch for a search.
+    """
+
+    def __init__(
+        self, api_key: str, contract_id: Optional[str] = None, top_n: int = 10
+    ):
+        from dria import Dria, Models
```

Comment:
can we try/except this import with a nice error message telling you how to install it